docs/source-pytorch/cli/lightning_cli.rst

```diff
@@ -60,34 +76,52 @@ Advanced use
 .. displayitem::
    :header: YAML for production
    :description: Use the Lightning CLI with YAMLs for production environments
-   :col_css: col-md-6
+   :col_css: col-md-4
    :button_link: lightning_cli_advanced_2.html
    :height: 150
    :tag: advanced
 
 .. displayitem::
    :header: Customize for complex projects
-   :description: Learn how to implement CLIs for complex projects.
-   :col_css: col-md-6
+   :description: Learn how to implement CLIs for complex projects
+   :col_css: col-md-4
    :button_link: lightning_cli_advanced_3.html
    :height: 150
-   :tag: expert
+   :tag: advanced
 
 .. displayitem::
    :header: Extend the Lightning CLI
    :description: Customize the Lightning CLI
-   :col_css: col-md-6
+   :col_css: col-md-4
    :button_link: lightning_cli_expert.html
    :height: 150
    :tag: expert
 
+----
+
+*************
+Miscellaneous
+*************
+
+.. raw:: html
+
+    <div class="display-card-container">
+        <div class="row">
+
 .. displayitem::
    :header: FAQ
    :description: Frequently asked questions about working with the Lightning CLI and YAML files
    :col_css: col-md-6
    :button_link: lightning_cli_faq.html
    :height: 150
 
+.. displayitem::
+   :header: Legacy CLIs
+   :description: Documentation for the legacy argparse-based CLIs
+   :col_css: col-md-6
+   :button_link: ../common/hyperparameters.html
+   :height: 150
```

Comment:
@awaelchli The CLI was recently promoted out of utilities into `pytorch_lightning.cli`. There's no advantage in the old functionality compared to the CLI. If legacy does not fit right with you (somewhat true since we haven't deprecated it), then maybe "old" is a better term.

Reply:
"legacy" was just a suggestion. Could be some other word or just "argparse-based CLIs". My main goal was not to deprecate or demote it. It is just so that people new to lightning discover first `LightningCLI` and avoid them implementing things that wouldn't be required if they use `LightningCLI`.

Reply:
@carmocca I am just surprised that we are calling it legacy and old, despite Python having the argparse library built-in and it's not going to disappear. No matter how awesome jsonargparse and LightningCLI are right now, there will be people who want to migrate to Lightning and just take their existing argparse code with them. They want to parse them and pass them into the model manually. To call this legacy or to even deprecate it (how even?) is a bit strong IMO. 

I suggest to recommend in the docs, in the hyperparameters.rst for example, whenever we can, to use the LightningCLI. This will promote the LightningCLI and still make it clear that you can use whatever.

Reply:
`argparse` itself is not legacy/old, but our `from_argparse_args` logic is as it has many limitations that the CLI does not. We will always allow the user to use whatever, but we are no longer investing resources into the pure argparse functionality ([example](https://github.com/Lightning-AI/lightning/issues/12974))

Reply:
Any suggestions on what I should do? Having both the `LightningCLI` and the `from_argparse_args` documentations under a single CLIs section is okay? If so, what should be the main title and the titles of the subsections?

Reply:
Feel free to resolve my comment here. I don't have a strong opinion on this anymore and it's probably ok. I don't have a proposal. I just wish the hyperparameters.rst would not mix argparsing and the save_hyperparameters logic, because that one is definitely not legacy.

Reply:
That sounds good to me. I can move it in this pull request. How about a "Save hyperparameters" section in https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html?

Reply:
Moved the save_hyperparameters docs to the lightning_module page.